templates/migrate/phobos_01_create_events.rb.erb

```diff
@@ -1,6 +1,7 @@
 class Phobos01CreateEvents < ActiveRecord::Migration[<%= ActiveRecord::Migration.current_version %>]
   def up
     create_table :phobos_db_checkpoint_events do |t|
+      t.timestamp :created_at,    index: false
```

Comment:
Normally it should not be needed as it is never added by default. Also, since we do not query by it (only order by it) we do not introduce the need of an index.

Let us add an index later, if needed.